x-pack/solutions/security/test/security_solution_api_integration/config/serverless/config.base.essentials.ts

```diff
@@ -21,6 +21,7 @@ export interface CreateTestConfigOptions {
    * log level. If needed, you can adjust the logging level via `kbnTestServer.serverArgs`.
    */
   kbnTestServerWait?: RegExp;
+  indexRefreshInterval?: string | false;
```

Comment:
nit: Adding a comment explaining the purpose and giving string value examples would be great. And looks like this comment should be added to each `indexRefreshInterval` param definition.